hwdb.d/60-input-id.hwdb

```diff
@@ -79,3 +79,9 @@ id-input:modalias:input:b0003v068Ep00F2*
 # TEX Shinobi Trackpoint
 id-input:modalias:input:b0003v04D9p0407e0111-e0,1,2,4*
  ID_INPUT_POINTINGSTICK=1
+
+# ASRock LED Controller
+id-input:modalias:input:b0003v26CEp01A2*
+ ID_INPUT_JOYSTICK=
+ ID_INPUT_KEY=
```

Comment:
Let me confirm that the device has no real keys, right?

Reply:
The device has no real keys. The devices is controlled by i2c interface and some settings in UEFI, and it provides an header to connect LED strips and similar devices.

I suppose it's possible that ASRock intended to connect devices with buttons for controlling LEDs to it, but: (i) the controller itself does not have key, (ii) to my knowledge no such device exists. So I think we can unset that property as well.

On a sidenote, unsetting those properties does not affect the i2c interface, [OpenRGB](https://gitlab.com/CalcProgrammer1/OpenRGB) still interacts normally with the device.